EventFilter/L1TRawToDigi/plugins/implementations_stage2/TauUnpacker.h

```diff
@@ -7,7 +7,16 @@ namespace l1t {
    namespace stage2 {
       class TauUnpacker : public Unpacker {
          public:
+            TauUnpacker();
+            ~TauUnpacker() override {};
+
             bool unpack(const Block& block, UnpackerCollections *coll) override;
+
+            inline void setTauCopy(const unsigned int copy) { TauCopy_ = copy; };
+
+         private:
+            TauBxCollection* res_;
```

Comment:
res_ is not used since a variable with the same name is also declared in the .cc. This one can be removed.